indico/modules/events/abstracts/controllers/reviewing.py

```diff
@@ -278,7 +278,7 @@ class RHDisplayAbstractsExportExcel(AbstractsExportExcel, RHDisplayAbstractsActi
 
 class RHEditReviewedForTrackList(RHAbstractBase):
     def _check_abstract_protection(self):
-        return self.abstract.can_judge(session.user, check_state=True)
+        return self.event.cfa.allow_convener_judgment and self.abstract.can_convene(session.user, check_state=True)
```

Comment:
Don't you want to use the new `can_change_tracks` method here? :D

Reply:
🙈